packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -154,7 +171,11 @@ class PrerenderDataExtractPlugin {
 		this.data_ = JSON.stringify(cliData.preRenderData || {});
 	}
 	apply(compiler) {
-		compiler.hooks.emit.tap('PrerenderDataExtractPlugin', compilation => {
+		compiler.hooks.emit.tap('PrerenderDataExtractPlugin', (compilation) => {
+			if (this.location_ === `${PREACT_FALLBACK_URL}/`) {
```

Comment:
Wondering if we still need this with the custom 200 option.

Reply:
Yes, this makes sure that the fallback page has no pre-render data. 
Thus in this case the pre-render data will always rely on the current URL .